### PR TITLE
RWKV-7 fixes

### PR DESCRIPTION
**Pull request by Triang-jyed-driung**
1. Implemented attention mask for Channel-mixing module
2. Fixed returning `None` for logits
3. Always launch `chunk` kernel if training

### CODE REVIEW

**Comment by yzhangcs** [comment]
Cool! Thank you.
